content/general/authentication.textile

```diff
@@ -84,26 +84,26 @@ Whilst Basic Authentication is simple, we recommend it to be only used on the se
 
 h3(#token-authentication). Token Authentication explained
 
-Token authentication is the recommended authentication scheme for client devices as tokens are short-lived and may more readily be distributed to clients where there is a risk of compromise. Tokens may also be issued with a particular scope - such as a limited set of "access rights or capabilities":#capabilities-explained or be limited to a "specific identity (<span lang="default">@clientId@</span><span lang="csharp">@ClientId@</span>)":#identified-clients. Tokens are most commonly issued server-side and may be signed using a private API key which guarantees to Ably that the token request is authentic and is issued by a trusted issuer.
+Token authentication is the recommended authentication scheme for client devices as tokens are short-lived and may more readily be distributed to clients where there is a risk of compromise. Ably-compatible tokens may also be issued with a particular scope - such as a limited set of "access rights or capabilities":#capabilities-explained or be limited to a "specific identity (<span lang="default">@clientId@</span><span lang="csharp">@ClientId@</span>)":#identified-clients. Ably-compatible tokens are most commonly issued server-side, although the type of Ably-compatible token issued varies. At present, Ably supports "Ably Tokens":#token-process, "Ably TokenRequests":#token-request-process, "JWT with an embedded Ably Token":#jwt-embed-process, and "Ably JWT":#ably-jwt-process.
```

Comment:
I don't understand why "Ably TokenRequests" is mentioned in the final sentence here. The other listed things are kinds of token.